pkg/module_manager/apis/v1alpha1/module.go

```diff
@@ -0,0 +1,82 @@
+package v1alpha1
+
+import (
+	"strings"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+// +genclient
+// +genclient:nonNamespaced
+// +k8s:deepcopy-gen=true
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// Module kubernetes object
+type Module struct {
+	metav1.TypeMeta `json:",inline"`
+	// Standard object's metadata.
+	// More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata
+	// +optional
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	Properties ModuleProperties `json:"properties"`
+
+	Status ModuleStatus `json:"status,omitempty"`
+}
+
+type ModuleProperties struct {
+	Weight int `json:"weight"`
+}
+
+type ModuleStatus struct{}
+
+type moduleKind struct{}
+
+func (ms *ModuleStatus) GetObjectKind() schema.ObjectKind {
+	return &moduleKind{}
+}
+
+func (mk *moduleKind) SetGroupVersionKind(_ schema.GroupVersionKind) {}
+func (mk *moduleKind) GroupVersionKind() schema.GroupVersionKind {
+	return ModuleGVK
+}
+
+var ModuleGVK = schema.GroupVersionKind{Group: "deckhouse.io", Version: "v1alpha1", Kind: "Module"}
```

Comment:
```suggestion
var ModuleGVK = schema.GroupVersionKind{Group: "deckhouse.io", Version: "v1alpha1", Kind: "Module"}
```
Don't think it should be a deckhouse. It would be better to pass GVR as a parameter.

Reply:
Removed to the Dechkouse